rambo/Vagrantfile

```diff
@@ -67,18 +67,20 @@ unless (hosts.include? VM_HOST)
    abort("ABORTED - VM host not in host list. Did you have a typo?")
 end
 
-sizes = ['512mb',
-         '1024mb',
-         '2048mb',
-        ]
+sizes = {'512' => '20gb',
+         '1024' => '30gb',
+         '2048' => '40gb',
+        }
 
-VM_SIZE = if ENV["VMSIZE"]
+VM_RAM = if ENV["VMSIZE"] and sizes.has_key? :ENV["VMSIZE"]
```

Comment:
Please change the following var names:

* ENV["VMSIZE"] -> ENV["RAMBO_RAM"]
* VM_HARDDRIVE -> ENV["RAMBO_DRIVESIZE"] (note this makes it an env var)

Then, drop usage of what is currently VM_RAM in favor of using ENV["RAMBO_RAM"] directly. With this change I think you can simplify the logic to just be

```
ENV["RAMBO_RAM"] = if not ENV["RAMBO_RAM"]
                    '2048'
                  end
```


After this I'll make another follow-up ticket for making a couple Ruby util functions similar to https://github.com/terminal-labs/rambo/blob/master/rambo/utils.py#L16-L24 to make this not hardcoded and prettier.